examples/blobfuse/config.json

```diff
@@ -0,0 +1,72 @@
+{
+  "location": "variables.location",
+  "resource_group": "variables.resource_group",
+  "install_from": "headnode",
+  "admin_user": "variables.ad_admin",
+  "variables": {
+    "image": "OpenLogic:CentOS:7.6:latest",
+    "hpc_image": "OpenLogic:CentOS-HPC:7.6:latest",
+    "location": "<NOT-SET>",
+    "resource_group": "<NOT-SET>",
+    "vnet_resource_group": "variables.resource_group",
+    "blob_account": "<NOT-SET>",
+    "blob_container": "<NOT-SET>",
+    "blob_key": "<NOT-SET>",
+    "blob_mount": "/blob"
+  },
+  "vnet": {
+    "resource_group": "variables.vnet_resource_group",
+    "name": "hpcvnet",
+    "address_prefix": "10.2.0.0/20",
+    "subnets": {
+      "compute": "10.2.4.0/22"
+    }
+  },
+  "resources": {
+    "headnode": {
+      "type": "vm",
+      "vm_type": "Standard_HC44rs",
```

Comment:
Please stop using HC when not needed. Use Standard_D8s_v3 which is enough for this kind of testing